lib/lev/status.rb

```diff
@@ -96,13 +145,13 @@ def has_reserved_keys?(hash)
     end
 
     def push(key, new_item)
-      new_value = (get(key) || []).push(new_item)
+      new_value = (send(key) || []).push(new_item)
       set(key => new_value)
     end
 
     STATES.each do |state|
       define_method("#{state}?") do
-        get('state') == state
+        status == state
```

Comment:
Decided to name it BackgroundJob (and BlackHoleStatus is NoBackgroundJob (not as sexy))